gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
Did anything change in the wrapper for 6.4 -> 6.5? Don't see anything checked in PR
https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
I checked by running `./gradlew wrapper --gradle-version 6.5` command to upgrade the wrapper, and nothing got changed.